src/state/get-droppable-over.js

```diff
@@ -1,23 +1,60 @@
 // @flow
-import type { Position, Rect } from 'css-box-model';
+import type { Position, Rect, BoxModel } from 'css-box-model';
 import { toDroppableList } from './dimension-structures';
-import isPositionInFrame from './visibility/is-position-in-frame';
-import { find } from '../native-with-fallback';
+import { distance } from './position';
+import isWithin from './is-within';
+
 import type {
+  DroppableId,
   DroppableDimension,
+  DraggableDimension,
   DroppableDimensionMap,
-  DroppableId,
 } from '../types';
 
+type isCandidateArgs = {|
+  active: Rect,
+  target: Position,
+  client: BoxModel,
+|};
+
+const isCandidate = ({ active, target, client }: isCandidateArgs) => {
+  const isWithinVertical = isWithin(active.top, active.bottom)(target.y);
+  const isWithinHorizontal = isWithin(active.left, active.right)(target.x);
+  const { width: draggableWidth } = client.borderBox;
+  const activeCenterHorizontal = active.center.x;
+  const isCollidingLeft =
+    activeCenterHorizontal > target.x - draggableWidth / 2;
+  const isCollidingRight =
+    activeCenterHorizontal < target.x + draggableWidth / 2;
+
+  // TODO: shrink active
+
+  // TODO: current draggable rect have any overlap with smaller active
+
+  // Has one of the edges passed a center point
+
+  // TODO: check vertical cases
```

Comment:
These two checks dont consider vertical cases, where a `draggable` is being moved upward from another `draggable` below it.